src/repositories/prisma/project-repository.ts

```diff
@@ -0,0 +1,9 @@
+import { Prisma, Project } from '@prisma/client'
+
+export interface ProjectRepository {
+  create(
+    data: Prisma.ProjectCreateWithoutUserInput,
```

Comment:
It might be better to use the UncheckedCreateInput types from Prisma. It's the responsibility of the use case to verify if the user exists.